docs/graphql-admin/locales.mdx

```diff
@@ -0,0 +1,256 @@
+# GraphQL Admin: Locale overview
+
+## Introduction
+
+The BigCommerce GraphQL Admin API provides the same access to the locales field as the REST Management [Store Locale](/docs/rest-management/settings/store-locale) API. Having access to locale data in the graph alongside related data makes it easier to build headless storefronts.
+
+## Getting started
+
+The GraphQL Admin API's allows merchants to add and remove locales per each sales channel. 
+
+## Example requests
+This section contains examples of GraphQL Admin API local mutations that you can send with the following HTTP configuration.
+
+```http filename="GraphQL Admin API HTTP configuration" showLineNumbers copy
+POST https://api.bigcommerce.com/accounts/{{store_hash}}/graphql
+X-Auth-Token: {{access_token}}
+Accept: application/json
+Content-Type: application/json
+```
+### Query channel locales
+
+The following query returns channel locales. 
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Example query: Get locales" showLineNumbers copy
+    query {
+      store {
+        locales(input: { channelId: "bc/store/channel/1"}) {
+          edges {
+            node {
+              code
+              status
+              isDefault
+            }
+          }
+        }
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+
+    ```json filename="Example query: Get locales" showLineNumbers copy
+     {
+      "data": {
+        "store": {
+          "locales": 
+            "edges": [
+              {
+                "node": {
+                  "code": "en",
+                  "status": "INACTIVE",
+                  "isDefault": true
+                }
+              }
+            ]
+          }
+        }
+      }
+    ```
+  </Tab>
+</Tabs>
+
+### Query locales supported by the platform
```

Comment:
Let's put this one on top, as this is kind of store level query which allows to query all the locales supported by BC. 
The above query for channel locales is operating in a context of a channel and it related to mutation below. 
Let's group it this way.